addons/website_forum/controllers/main.py

```diff
@@ -479,24 +479,24 @@ def open_user(self, forum, user_id=0, **post):
         Followers = request.env['mail.followers']
         Data = request.env["ir.model.data"]
 
-        user = User.sudo().search([('id', '=', user_id)])
+        selected_user = User.sudo().search([('id', '=', user_id)])
```

Comment:
Reduce the diff, remove unless rename. Just keep the change for the return key ('user' into 'selected_user').